packages/nlp/nlp.0.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "Dave Tucker <dave@dtucker.co.uk>"
+authors: ["Dave Tucker <dave@dtucker.co.uk>"]
+homepage: "https://github.com/dave-tucker/ocaml-nlp"
+doc: "https://dave-tucker.github.io/ocaml-nlp/doc"
+license: "ISC"
+dev-repo: "https://github.com/dave-tucker/ocaml-nlp.git"
+bug-reports: "https://github.com/dave-tucker/ocaml-nlp/issues"
+tags: []
+available: [ ocaml-version >= "4.01.0"]
+depends: [
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build} 
+  "astring"]
+depopts: []
+build: [
+  "ocaml" "pkg/pkg.ml" "build"
+          "--pinned" pinned ]
```

Comment:
Because of the need to support OPAM 1.2.0 (distributed in Debian stable), we need to quote the `pinned` macro.  If you make that `"--pinned" "%{pinned}%"` then it will pass CI for OPAM 1.2.0 as well.